fact-bounty-client/src/pages/main/components/Post.js

```diff
@@ -43,11 +40,6 @@ const styles = {
 }
 
 class Post extends Component {
-
-	constructor(props) {
```

Comment:
This function should not be removed

Reply:
I think the constructor was not serving any purpose and the compiler was throwing a warning of useless constructor